src/Pages/Main/Components/StoreDetail/Info.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { BiChevronDown, BiChevronUp } from 'react-icons/bi';
+import StoreInfo from './StoreInfo';
+import Map from './Map';
+import Menu from './Menu';
+import styled from 'styled-components';
+
+const Info = props => {
+  const [isActive, setActive] = useState(false);
+  const activeToggle = () => {
+    setActive(!isActive);
+  };
+
+  const INFO = [
+    {
+      id: 1,
+      title: '가게 정보',
+      content: <StoreInfo address={props.address} />,
+    },
+    {
+      id: 2,
+      title: '메뉴',
+      content: (
+        <Content>
+          <Menu />
+        </Content>
+      ),
+    },
+    {
+      id: 3,
+      title: '위치',
+      content: (
+        <Content>
+          {props.address}
+          <Map openRoadview={props.openRoadview} />
+        </Content>
+      ),
+    },
+    {
+      id: 4,
+      title: '인근 전철역',
+      content: <Content>{props.metro_stations.toString()}</Content>,
+    },
+    {
+      id: 5,
+      title: '리뷰',
+      content: <Content>{props.description}</Content>,
+    },
+  ];
+
+  return (
+    <>
+      {INFO.map(data => (
+        <Infomation key={data.id}>
+          <Toggle>
+            {data.title}
+            <Button onClick={activeToggle}>
+              {!isActive ? BiChevronUp() : BiChevronDown()}
+            </Button>
+          </Toggle>
+          {!isActive && data.content}
+        </Infomation>
+      ))}
+    </>
+  );
+};
```

Comment:
반복을 줄이고 있는 것 같지가 않아요...
```suggestion
  const INFO = [
    {
      id: 1,
      title: '가게 정보',
      content: <StoreInfo address={props.address} />,
    },
    {
      id: 2,
      title: '메뉴',
      content: (
        <Content>
          <Menu />
        </Content>
      ),
    },
    {
      id: 3,
      title: '위치',
      content: (
        <Content>
          {props.address}
          <Map openRoadview={props.openRoadview} />
        </Content>
      ),
    },
    {
      id: 4,
      title: '인근 전철역',
      content: <Content>{props.metro_stations.toString()}</Content>,
    },
    {
      id: 5,
      title: '리뷰',
      content: <Content>{props.description}</Content>,
    },
  ];

  return (
    <>
      {INFO.map(data => (
        <Infomation key={data.id}>
          <Toggle>
            {data.title}
            <Button onClick={activeToggle}>
              {!isActive ? BiChevronUp() : BiChevronDown()}
            </Button>
          </Toggle>
          {!isActive && data.content}
        </Infomation>
      ))}
    </>
  );
};
```